app/services/caesar_importer.rb

```diff
@@ -0,0 +1,77 @@
+class CaesarImporter
+  attr_reader :reduction_id, :reducible, :data, :subject_info
+
+  def initialize(**args)
+    @reduction_id = args[:id]
+    @reducible = args[:reducible]
+    @data = args[:data]
+    @subject_info = args[:subject]
+  end
+
+  def process
+    generate_parent_resources
+
+    # If a transcription with this id already exists, an exception will be raised
+    new_transcription = Transcription.new(transcription_create_attrs)
+    new_transcription.workflow = workflow
+    new_transcription.save!
+  end
+
+  private
+
+  def workflow
+    Workflow.find_by(id: reducible[:id])
+  end
+
+  def generate_parent_resources
+    # If the workflow exists, the project does too
+    unless workflow
+      response = pull_wf_and_project
+      existing_project = Project.find_by(id: response[:project][:id])
```

Comment:
Might be able to use `Project.find_or_initialize_by` to clean up some code

Reply:
I think I like the discrete `create!`s tbh. Using FOIB means that the object (`existing_project`) always actually exists (even if it hasn't been persisted) , so I can't use `if existing_project` as a conditional, so I'd have to ask about its state anyway.